src/main/java/eu/darkbot/api/entities/utils/Area.java

```diff
@@ -0,0 +1,8 @@
+package eu.darkbot.api.entities.utils;
+
+public class Area extends com.github.manolo8.darkbot.core.utils.pathfinder.Area {
```

Comment:
Keep Area as an interface, and `AreaImpl extends manolo8.Area implements Area`. In the future we will probably want circular areas or other kind of implementations that don't necessarily conform to the minxy maxxy rectangle model